### ISSUE TITLE
object alt text on bilingual pages

### ISSUE THREAD

**Issue by dalhosny** [wet-boew/GCWeb#1242]
Hello,

We noticed on the example bilingual server message page, some of the `<object>` alt text defined in the `aria-label `is not bilingual.  

Ie.
http://wet-boew.github.io/themes-dist/GCWeb/servermessage-en-fr.html

```

<div class="col-sm-6">
<object id="gcwu-sig" type="image/svg+xml" tabindex="-1" role="img" data="./GCWeb/assets/sig-blk-en.svg" aria-label="Government of Canada"></object>
</div>
<div class="col-sm-6">
<object id="wmms" type="image/svg+xml" tabindex="-1" role="img" data="./GCWeb/assets/wmms-blk.svg" aria-label="Symbol of the Government of Canada"></object>
</div>

```

We were also discussing which of these WCAG techniques would be better for the alt text, as H53 would allow us to wrap the French text in a `<span lang="fr"> `to allow screen readers to pronounce it correctly:

http://www.w3.org/TR/WCAG20-TECHS/ARIA6.html
http://www.w3.org/TR/WCAG20-TECHS/H53.html

What do you recommend?

Thank you.

**Comment by EricDunsworth** [comment]
@dalhosny FYI #1241 addresses the bilingualism issue you identified in the FIP image. Apparently I sent it out literally a minute before this issue was opened lol.

**Comment by dalhosny** [comment]
Well that's convenient, thank you!

I was looking at the change in #1241 though and it seems to be following this WCAG technique:

http://www.w3.org/TR/WCAG20-TECHS/H2.html

That works for any `<object>` contained in an `<a>`, but can you recommend an approach when no` <a>` is used/required?  I think we'll need another way to associate the `<object>` with the text.  For example, the wmms-blk.svg in my previous post.

Thank you.

**Comment by EricDunsworth** [comment]
@dalhosny In #1241, all I did was modify the content of pre-existing text alternatives to be more consistent with the text in the SVGs. I hasn't considered changing any surrounding HTML markup.

From what I can tell, all of WET's unlinked SVG objects are currently employing [technique G94](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G94) and [technique ARIA6](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/ARIA6) to comply with [success criterion 1.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html).

I really like your suggestion for applying [technique H53](https://www.w3.org/TR/WCAG20-TECHS/H53.html) for bilingual alt text since it'd make it possible to programmatically declare the "opposite" languages in the text alternatives.

Before proceeding with implementing that kind of technique, we might want to consider the following:

1. What the scope of this change should be (i.e. only bilingual SVGs or all SVGs that warrant the use of text alternatives). I'd prefer doing it for all relevant SVGs for the sake of consistency.
2. If it's decided to implement that technique for all relevant SVGs, which WET repos should get it? IMO only GCWeb, theme-gcwu-fegc and theme-gc-intranet should be in scope. I don't believe any other WET repos are using SVGs to represent images of text.
3. Whether this kind of change is appropriate for a 4.0.x maintenance release. I personally think it would be. Even though it's an HTML markup change, it isn't a breaking change and it doesn't look like WET 4.1/5.0 are coming out anytime soon.

@LaurentGoderre @lucas-hay @shawnthompson @nschonni Thoughts?
